### ISSUE TITLE
[Suggestion] Baby player trophy that can have different skins.

### ISSUE THREAD

**Issue by pantsguyapp** [GizmoTheMoonPig/OpenBlocksTrophies#26]
### The Suggestion

This trophy display the default steve model and skin but in baby mode (bigger head than body). The trophy can be renamed to take on other people's skins. The skin renaming can be done via an anvil.

This idea was taken from the small gifter statues in Vault Hunters.

### Why Should It Be in the Mod?

Because there has been a trophy for every other mobs but not the player, and the ability for the trophy to have different skins is just a plus.

### Please Read and Confirm the Following

- [X] I have confirmed this suggestion is unique and has not been suggested already.

**Comment by GizmoTheMoonPig** [comment]
I like the idea. Since you're suggesting it, how would you imagine it drops? Is it from killing players, or do you get it a different way?
I dont know if im particularly fond of the bigger head thing or if I can even do that without a custom entity model. I'll have to look into it at some point.

**Comment by pantsguyapp** [comment]
For starter, I think we can find them in vanilla structure loots as a hint to ancient people artifact, maybe from archeology also. 
For renewability, I'd say yes, they do drop from a player being killed by another player, followed by the same drop ratio, in singleplayer, I'd suggest the trophy is dropped when the player is killed by a mob, and not by the environment, althought this will encourage self farming method for these, but thats all I can come up with.
The bigger head thing is supposed to add in a uniqueness to the trophy, rather than just look like a rendered model like every other mobs, the player trophy is unique, special and most importantly, cuter.
The last info I want to say as you are interested in the idea is: Try to look into the gift statues in Vaul Hunter and compare the base with the actual model, maybe see if you can duplicate the ratio of the statue also, I'll let it up to you.

**Comment by pantsguyapp** [comment]
Wow thanks, I nevee thought that this will be a good idea.
Sure, heres my idea.
Obtain method:
- Via archeology and structure loots, this given some lore about ancient people used these ritualistically, can scrap if you think this make people think of human sacrifice.
- From players being killed, with the same drop rate of other trophies, when killed by another player in online, or killed by a creeper type mob for offline, creepers are picked because they essentially die when they kill you, making player trophies incredibly hard to farm in ssp.

Extra note: I think the bigger head invoke a bubblehead figurine image and its kind of cute and funny, thats why I brought it up. Please consult the small gifter statues from 1.16 Vault Hunters, as the size and model is how I imagine the trophy would be. If thats not good in your eyes, then you can do whatever. 
About the name changing thing, I think they should require name tags, as that will make it more difficult, and when named, the name should pop up as name when pointing at the trophy.

That is all.

**Comment by GizmoTheMoonPig** [comment]
Started the trophies! Still got a couple kinks to iron out but they work! 
I'll keep the normal trophy drop requirements like other trophies have, but I think instead of normal creepers I'll make it a charged drop (chance of course, I don't like making things easy to get)
![2023-11-27_00 49 37](https://github.com/GizmoTheMoonPig/OpenBlocksTrophies/assets/67468252/9414ed04-33b0-415d-b0b7-43ccc121da87)

**Comment by pantsguyapp** [comment]
I am SOOOO in love with them. Meh they're fine, they can be an Easter Egg even. But do you think you can make the ign name of the player the statue is using the skin appear when you look at them? Also which method did you pick for naming? Anvil or name tags? Name tags require an extra step to do so.

**Comment by GizmoTheMoonPig** [comment]
gonna stick with anvils for simplicity's sake. Rendering names above them shouldn't be too hard, i'll look into it tomorrow. Its getting late here :P 
Thanks for this suggestion. Ive been thinking about player trophies for a while actually, but you pushed me to finally get to it, and ive had a lot of fun implementing them so far

**Comment by pantsguyapp** [comment]
But still, only obtainable from charged creepers right?

**Comment by pantsguyapp** [comment]
One last note, there need to be a way to delete the skin and revert back to an unamed trophy, can you add a crafting recipe for named player trophies to revert back to unamed in a crafting grid?

**Comment by GizmoTheMoonPig** [comment]
removing the name from the trophy in an anvil removes the name completely, reverting the name back to "Player Trophy" and changing the skin back to steve. 
Slim skins do work, yes. Just so happens that all the TF dev skins use the normal arms lmao, heres an image with some more skins of some of my friends. A few have slim arms so you can see they work
![2023-11-27_10 07 31](https://github.com/GizmoTheMoonPig/OpenBlocksTrophies/assets/67468252/51254e96-fc4a-4a2e-9c20-27775a80ddf9)

**Comment by pantsguyapp** [comment]
Oh btw what happened if the player themselves change skin, does the trophy itself detect the change and automatically apply new skin upon relogging in?

**Comment by GizmoTheMoonPig** [comment]
yeah, since its based off the actual game profile from the mojang servers, whatever skin the player is currently wearing will be used. It'll only change on game reload as I cache the game profiles to improve performance

**Comment by pantsguyapp** [comment]
One last note, do you think you can increase the size of the base model just a little bit, the gifter statues in VH has bigger base than the trophy.
Have a look.
![285951924-51254e96-fc4a-4a2e-9c20-27775a80ddf9](https://github.com/GizmoTheMoonPig/OpenBlocksTrophies/assets/92203672/396feb2c-b09e-4e1d-9326-9284258466de)

![tpejs30g66d91](https://github.com/GizmoTheMoonPig/OpenBlocksTrophies/assets/92203672/f22770fa-90f0-43be-bb23-0f91913d417d)

**Comment by pantsguyapp** [comment]
So size will be keep the same then.

**Comment by GizmoTheMoonPig** [comment]
oh whoops, missed that comment. 
I'm gonna keep the trophy size the way it is now so it matches with other trophies. If I make them any bigger they start to stretch out of your inventory slot

**Comment by pantsguyapp** [comment]
Very well friend, I am just graceful for my suggestion to actually being implemented ;)

**Comment by GizmoTheMoonPig** [comment]
thanks for sticking through this with me! I'll have an update on curseforge shortly with the players and all of the other changes ive done. Hope you enjoy!

**Comment by pantsguyapp** [comment]
Since this was inspired by a feature that used to be in Vault Hunters lets try to convince them to include this mod when the pack is ported to 1.19. In exchange, do you think it is wise to add in compat for the vault mod in the future?

**Comment by pantsguyapp** [comment]
Sorry for bothering again, but since this trophy is unlike others, can you put in a hint on how to get them via the advancements, something down the line of "A Creeper Is You." With the description of "Successfully hugged a creeper and got a player trophy?"
